mtags/src/main/scala-2/scala/meta/internal/pc/completions/ArgCompletions.scala

```diff
@@ -169,6 +170,18 @@ trait ArgCompletions { this: MetalsGlobal =>
       }
     }
 
+    private def getParamsFromObject(): List[Symbol] = {
+      methodSym.info.members
+        .collect {
+          case m
+              if m.decodedName == "apply" &&
+                m.paramss.flatten.length >= apply.args.length =>
+            m.paramss.flatten
```

Comment:
We can do it in another PR for sure, but we should probably also match it by names/types that already exist in the apply